### PR TITLE
ywca-4: Custom Title 

### PR DESCRIPTION
**Pull request by ArielleMurad**

### CODE REVIEW

**Comment by sumeetgrewal** [comment]
Please pull from master

File: assets/css/ywca.css [hunk 1]
```diff
@@ -14,6 +14,23 @@ Blue - #6399AB
 @font-face { font-family: Gotham; font-style: italic ; src: url('../fonts/Gotham-BookItalic.otf');}
 
 body {
+<<<<<<< HEAD
```
Comment (sumeetgrewal) [inline assets/css/ywca.css]: if this merge conflict is fixed please remove all conflict markers

File: custom_title_field.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<?php
```
Comment (sumeetgrewal) [inline custom_title_field.php]: rename to title.php

File: custom_title_field.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+			),
+			'default_value' => '',
+			'tabs' => 'all',
+			'toolbar' => 'full',
+			'media_upload' => 1,
+			'delay' => 0,
+		),
+	),
+	'location' => array(
+		array(
```
Comment (sumeetgrewal) [inline custom_title_field.php]: remove extra nested arrays

File: functions.php [hunk 4]
```diff
@@ -11,7 +17,9 @@ function ywca_enqueue_styles() {
     wp_enqueue_style('customcss', get_template_directory_uri() . '/assets/css/ywca.css', array(), '1.0', 'all');
     wp_enqueue_script('customjs', get_template_directory_uri() . '/assets/js/ywca.js', array(), '1.0.0', true);
 }
-add_action( 'wp_enqueue_scripts', 'bootstrap_enqueue_styles' );
+
+add_action( 'wp_enqueue_scripts', 'bootstrapstarter_enqueue_styles' );
+add_action( 'wp_enqueue_scripts', 'bootstrapstarter_enqueue_scripts' );
```
Comment (sumeetgrewal) [inline functions.php]: use functions.php from master branch, these are the wrong enqueue scripts

File: index.php [hunk 5]
```diff
@@ -1,20 +1,28 @@
...
-</a>
+
+<?php
+if (have_posts()) {
+    while (have_posts()) {
+        the_post();
+        ?>
+        <article class="post">
+        <p><?php the_content(); ?></p>
+    </article>
```
Comment (sumeetgrewal) [inline index.php]: what is the article class for?

File: page-registration.php [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        <article class="post">
+        <p><?php the_content(); ?></p>
+    </article>
+    <?php
+} // end while
+} // end if
+?>
+
+<!-- <h5><?php the_date(); ?></h5> -->
+
```
Comment (sumeetgrewal) [inline page-registration.php]: delete comment

File: custom_title_field.php (hunk 2 above)
Comment (ArielleMurad) [reply custom_title_field.php]: I already have a file named title.php, should I rename it to something else?

Comment (sumeetgrewal) [reply custom_title_field.php]: That's okay, just use the template-parts and custom-field folders.

**Pull request merged**